### ISSUE TITLE
how to embed this loss function in keras or tensorflow?

### ISSUE THREAD

**Issue by shghahremani** [BelBES/mpl.pytorch#7]
Hi, thank you for the code.
how to embed this loss function in keras or tensorflow?

**Comment by bes-dev** [comment]
I think that you should write custom API of compute_weights function for keras/tf.
